.travis.yml

```
@@ -4,17 +4,13 @@ go_import_path: go.uber.org/multierr
 
 env:
   global:
-    - GO15VENDOREXPERIMENT=1
     - GO111MODULE=on
 
 go:
-  - 1.11.x
   - 1.12.x
   - 1.13.x
-
-cache:
-  directories:
-    - vendor
+  - 1.14.x
+  - 1.15.x
```

Since [gimme 1.5.4](https://github.com/travis-ci/gimme/releases/tag/v1.5.4), we've been able to use `stable` and `oldstable` instead of hard-coding the version numbers.

```suggestion
  - oldstable
  - stable
```